### ISSUE TITLE
how to use?

### ISSUE THREAD

**Issue by radioblahaj** [Sigton/messenger#1]

**Comment by Spookzy** [comment]
a lot of steps are required to fully set up the full project.
if we could. this project would probably be private, but we can't so it's public
github is just used for fast saving of this project

**Comment by radioblahaj** [comment]
you're not @Sigton

**Comment by Spookzy** [comment]
yes but i know how to use a lot of the project and usually work with Sigton on ideas

**Comment by Sigton** [comment]
Basically this program reads and writes to a .db file, (and unfortunately only works locally, but that was a requirement for our intended use) so if you have a database file on your system you can add that as a server in the servers dropdown.

Sigton
